analysis/src/CompletionBackEnd.ml

```diff
@@ -1157,6 +1157,17 @@ let completionsGetTypeEnv = function
   | {Completion.kind = Field ({typ}, _); env} :: _ -> Some (typ, env)
   | _ -> None
 
+let findReturnTypeOfFunctionAtLoc loc ~(env : QueryEnv.t) ~package ~debug =
+  match Cmt.loadFullCmtFromPath ~path:(env.file.uri |> Uri.toPath) with
```

Comment:
Since this seems to be intended for the current file,  we should have this info available since the `getCompletions` function. Perhaps pass around `~full` instead of `~package`, and when we get here it's already there.